src/FootPrintsInterpolator.cpp

```diff
@@ -35,7 +35,7 @@ bool FeetInterpolator::orderSteps()
     return true;
 }
 
-bool FeetInterpolator::createPhasesTimings()
+bool FeetInterpolator::createPhasesTimings(const InitialState &weightInLeftAtMergePoint)
```

Comment:
Since you are using only `initialVelocity` field of the `weightInLeftAtMergePoint` struct you may change the input variable of this method from 
```c++
const InitialState &weightInLeftAtMergePoint
```
 to 
```c++
const double &velocityWeightInLeftAtMergePoint
```
This allows to have a complete compatibility with [`dcmTrajectoryGenerator`](https://github.com/robotology/unicycle-footstep-planner/tree/dcmTrajectoryGenerator) branch.